receiver/awscontainerinsightreceiver/internal/stores/store.go

```diff
@@ -99,7 +99,6 @@ func (k *K8sDecorator) Decorate(metric *extractors.CAdvisorMetric) *extractors.C
 	}
 
 	AddKubernetesInfo(metric, kubernetesBlob, k.addContainerNameMetricLabel)
-	TagMetricSource(metric)
```

Comment:
talked about this w/Kulwant 1:1.  I am concerned about modifying the linux code, sometimes a subtle refactor can have unexpected negative impact.  He is going to change `TagMetricSource` to have a conditional to check for the scenario he is looking for and that will result in less code moving around

Reply:
Addressed this concern